stdlib/src/ssl.rs

```diff
@@ -992,6 +1021,33 @@ fn convert_ssl_error(
     vm.new_exception_msg(cls, msg.to_owned())
 }
 
+fn x509_stack_from_der(der: &[u8]) -> Result<Vec<X509>, ErrorStack> {
+    unsafe {
+        openssl::init();
```

Comment:
does `openssl::init()` is designed to be called whenever this function is called?

Reply:
Most of rust-openssl's functions call `init()` at the start of the function, e.g. `X509::stack_from_pem`'s first line calls it, since it's invalid to use the openssl api without calling it. Obviously we wouldn't get to the point since we call it in `make_module` but I figured since it's a more generic function it would be good to follow their convention.